getting_started/setup/setup_up_2_board.rst

```diff
@@ -28,10 +28,17 @@ will show UP-APL01 R3.6.
    
      this link needs to be updated to something accessible externally
 
-* Press F2 to enter BIOS main menu and check the BIOS version.
-* Press ENTER when prompted for password.
-* If the current BIOS version is older than 1.8, please update to 1.8
+* If the current BIOS version is older than 1.8, please update to v1.8
   before flashing v3.6.
+
+  .. note::
+
+    To check your BIOS version press
```

Comment:
@stripes416 Line 36: Need a carriage return after this as the interpreter is getting confused about what is intended. That's why this line is bolded when click "view file."

Reply:
And no need to indent 1) and 2). This should work:

  .. note::

    To check your BIOS version press

    1) DELETE or
    2) F7 and select 'Enter Setup'

Reply:
@intelkevinputnam --- I was wondering why that was in bold!